runtime/autoload/health/provider.vim

```diff
@@ -502,7 +502,17 @@ function! s:check_node() abort
           \ ['Install Node.js and verify that `node` and `npm` commands work.'])
     return
   endif
-  call health#report_info('Node: '. s:system('node -v'))
+  let node_v = s:system('node -v')
+  call health#report_info('Node: '. node_v)
+  if !s:shell_error
+    let node_v = split(split(node_v, "\n")[0][1:], '\.')
+    if node_v[0] < 6
+      call health#report_warn('node host does not support your node installation.')
+    endif
+  endif
+  if has('win32') || !provider#node#can_inspect()
+    call health#report_warn('$NVIM_NODE_HOST_DEBUG is no-op.')
```

Comment:
Users will be confused by this. How about:

    The nodejs version on this system does not support --inspect-brk, $NVIM_NODE_HOST_DEBUG will be ignored.